README.md

```diff
@@ -27,8 +27,8 @@ foreman-installer
 | Foreman Version | Katello Version | Plugin Version |
 | --------------- | --------------- | -------------- |
 | 3.3             | 4.5             | ~> 2.0.0       |
-| 3.1             | 4.3             | ~> 1.8.20\*    |
-| 3.0             | 4.2             | ~> 1.8.20      |
+| 3.1             | 4.3             | ~> 2.0.0       |
+| 3.0             | 4.2             | ~> 1.8.20\*    |
```

Comment:
This can be dropped. foreman_scc_manager 2.2.0 is supported starting in Foreman 3.3 . It is broken with Foreman 3.6, so we need to fix it there.

Reply:
I am unsure I understood you correctly. :see_no_evil:  Did my change make it better or worse?

Reply:
You can close the PR because with https://github.com/theforeman/foreman-packaging/pull/8928 this merged, the README is actually correct as it was :see_no_evil: